# generated by patch-package 6.5.0 on 2022-03-31 20:04:21
#
# command:
#   npx patch-package @emotion/react --exclude nothing
#
# declared package:
#   @emotion/react: 11.8.2
#
diff --git a/node_modules/@emotion/react/package.json b/node_modules/@emotion/react/package.json
index 22fba2d..fafe561 100644
--- a/node_modules/@emotion/react/package.json
+++ b/node_modules/@emotion/react/package.json
@@ -3,10 +3,6 @@
   "version": "11.8.2",
   "main": "dist/emotion-react.cjs.js",
   "module": "dist/emotion-react.esm.js",
-  "browser": {
-    "./dist/emotion-react.cjs.js": "./dist/emotion-react.browser.cjs.js",
-    "./dist/emotion-react.esm.js": "./dist/emotion-react.browser.esm.js"
-  },
   "types": "types/index.d.ts",
   "files": [
     "src",
